src/managed/CommonManaged.props

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
-    <VersionPrefix>2.0.2</VersionPrefix>
+    <VersionPrefix>2.0.3</VersionPrefix>
```

Comment:
I don't think we need to bump these unless we decide to ship one of these libraries.

Reply:
I added them because currently they are under the servicing list of packages. @eerhardt added them in this commit: https://github.com/dotnet/core-setup/commit/69f6efc7395c83721eab65fd30184a1bfb253ad3

Reply:
It was because of this change:

https://github.com/dotnet/core-setup/pull/3188

Reply:
Yes, we had to change PlatformAbstractions in order to support RHEL 6.

FYI - when dev/release/2.0.0 comes in, these version numbers will all roll up to the product-wide version number in the root.  See https://github.com/dotnet/core-setup/commit/df6ab9c2a48b47181bedfb2704888b074e8a269a#diff-f9d387bebca621498ce66c5220efb38dR25

Reply:
Makes sense. I will keep an eye on. Thanks for the heads up.

Reply:
OK keep these bumped given we need to ship them.